packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/FixedEnumProvider.cs

```diff
@@ -68,6 +71,38 @@ protected override IReadOnlyList<EnumTypeMember> BuildEnumValues()
                 var name = _isApiVersionEnum
                     ? inputValue.Name.ToApiVersionMemberName()
                     : inputValue.Name.ToCleanName();
+
+                // check if the enum member was renamed in custom code
+                string? customMemberName = null;
+                foreach (var customMember in customMembers)
+                {
+                    var attributes = customMember.Attributes;
+                    if (attributes == null)
+                    {
+                        continue;
+                    }
+
+                    foreach (var attribute in attributes)
+                    {
+                        if (CodeGenAttributes.TryGetCodeGenMemberAttributeValue(attribute, out var originalName)
+                            && originalName == name)
+                        {
+                            customMemberName = customMember.Name.ToCleanName();
```

Comment:
Why do we need the ToCleanName call? Since this comes from custom code I think we should use it as-is, but I assume ToCleanName would be a no-op here anyway.

Reply:
Fixed.